test/core/promise/try_concurrently_test.cc

```diff
@@ -29,38 +29,38 @@ class PromiseFactory {
  public:
   // Create a promise that resolves to Ok but has a memory allocation (to verify
   // destruction)
-  auto OkPromise(std::string tag) {
-    return [this, tag = std::move(tag),
+  auto OkPromise(absl::string_view tag) {
```

Comment:
Are generated strings always guaranteed to outlive the promise?

Reply:
added a comment... it puts a broader restriction than is necessary, but matches up well with the intent here.